check for Pillow attributes in different locations

dja001 opened the pull request:
In some versions Pillow, the attributes `PIL.Image.Transform` does not exist causing the extension to fail. 

With the changes proposed here, the extension will check `PERSPECTIVE` and `BICUBIC` in both:
```
PIL.Image.PERSPECTIVE
PIL.Image.BICUBIC
```
and
```
PIL.Image.Transform.PERSPECTIVE
PIL.Image.Resample.BICUBIC
```
and pick the first one it finds. 

This should enable this Inkscape extension to work seamlessly with different version of Pillow. 

fixes #7

s1291 commented:
Hi @dja001;
Thank you for your pull request.

I have intentionally made those changes in this commit https://github.com/s1291/InkRasterPerspective/commit/be869fb1b7abd837542595312f17280ed3191019
 because it was announced in pillow v9.1.0 that `PIL.Image. PERSPECTIVE` and `PIL.Image.BICUBIC` were deprecated and will be removed in v10.0 (See the release notes [here](https://pillow.readthedocs.io/en/stable/releasenotes/9.1.0.html#deprecations))

However, that decision was reversed in v9.4.0 and those constants are kept (See the note [here](https://pillow.readthedocs.io/en/stable/deprecations.html)).

Could you please push a pull request that reverts my commit above and adds comments about the deprecation of these constants in v9.1.0

Thank you

dja001 commented:
Good morning @s1291,  

Thanks, for digging up the story about the deprecation and de-deprecation of these constants. Managing changes in dependencies is not the most fun part of maintaining a project... 

In the latest commit of this PR, I followed your suggestions and added the story about these constants with links to the deprecation messages from the Pillow project. 

Since the constants are apparently there to stay, I also removed the logic looking for them in different places. As before be869fb, the code now calls `PIL.Image.PERSPECTIVE` and `PIL.Image.BICUBIC` directly. 

Take care,

s1291 commented:
@dja001 
Many thanks for your time.
